src/ensembl/src/ens-object/ensObjectActions.ts

```diff
@@ -82,7 +81,7 @@ const builtTrackList = (ensObject: EnsObjectResponse) => {
     description: ensObject.description,
     ensembl_object_id: ensObject.object_id, // we don't use this field
     label: ensObject.label,
-    track_id: 'gene-feat',
+    track_id: 'track:gene-feat',
```

Comment:
Same as the comment above about magic strings.

Reply:
That's done